### ISSUE TITLE
Train on custom dataset

### ISSUE THREAD

**Issue by d5423197** [Lee-Gihun/MEDIAR#20]
Hello there, 

I want to use this repo to train my own dataset. Which config file do you suggest me to start with? The baseline.json?

Thanks,

ZD

**Comment by Lee-Gihun** [comment]
Thank you for your interest in our work. The `baseline.json` represents the challenge baseline, so it is not our work. 

I suggest working directly on the final weights with small modifications to the finetuning configurations (e.g., `step2_finetuning/finetuning1.json`). 

You may want to change the learning rate to a smaller value.

**Comment by Lee-Gihun** [comment]
Please re-open this issue if you have further issues :)
